### ISSUE TITLE
No way of checking if ofxArtnetSender/Receiver is actually connected

### ISSUE THREAD

**Issue by bgstaal** [hiroyuki/ofxArtnet#25]
As of now there is no way of checking if Sender/Receiver is actually connected to the udp endpoint. This makes it impossible to check if one is actually connected before sending data.

**Comment by piotrus04** [comment]
Yes this is exactly the interest of Polling the receiver, but it seems that it is not implemented in this git

**Comment by hiroyuki** [comment]
I think you guys are talking about artpoll message.
It’s not implemented in this version. I don’t think it’s not that complicated though.

**Comment by adidahiya** [comment]
+1, I'm also interested in a method to check if the connection has been opened successfully

**Comment by piotrus04** [comment]
I found this from a friend of mine, 
https://github.com/mattbeghin/Art-Ext-Poll
and 
https://github.com/OpenLightingProject/artnet-examples/blob/master/src/artnet-test-master.c

but i don't have sufficient knowledge in C to make a good (coded well) implementation alone.

Somebody would be interested to implement it ?

**Comment by hiroyuki** [comment]
if you can wait some time, I will try it later. maybe in next month.

in case you can not wait, i think you can write your own code using ofxNetwork.
The basic infomation is [here](https://art-net.org.uk/structure/discovery-packets/).

**Comment by piotrus04** [comment]
Ok , i'll try. I can do it, it will just be quite dirty. If somebody can later clean the code... :)
I'll try to implement it in october.
thks
